components/blocks/testimonialsList.tsx

```diff
@@ -1,45 +1,27 @@
 import { useEffect, useState } from "react";
 import { FaSpinner } from "react-icons/fa";
 import type { Template } from "tinacms";
-import client from "../../.tina/__generated__/client";
+import { extractFileName } from "../../helpers/functions";
+import { getFilteredTestimonials } from "../../helpers/getTestimonials";
 import { TestimonialCard } from "./testimonialsCard";
 
 export const TestimonialsList = ({ data: { listOfCategoriesToHide = [] } }) => {
   const [testimonials, setTestimonials] = useState([]);
   const [hasLoaded, setHasLoaded] = useState(false);
 
-  const categoriesToHide = listOfCategoriesToHide?.map(
-    (category) => extractCategoryName(category?.categoryName)?.toLowerCase()
-  );
-
   useEffect(() => {
-    const loadTestimonials = () => {
-      client.queries.testimonialsConnection().then((data) => {
-        const testimonials = data.data?.testimonialsConnection?.edges?.map(
-          (edge) => edge?.node
-        );
+    const loadTestimonials = async () => {
+      const filteredTestimonials = await getFilteredTestimonials(
+        listOfCategoriesToHide
```

Comment:
given the method name - I would've expected that any param I would pass in would be to filter the return values to the ones I want to show, not hide - consider renaming the method name

Reply:
done - changed the method name to `getTestimonialsExcludingCategories`

Reply:
and variable name to `excludedCategories`